site/docs/v1/tech/tutorials/integrate-python-flask.md

```diff
@@ -0,0 +1,279 @@
+navcategory: getting-started
+prerequisites: python3 and pip3
+technology: Python Flask
+language: Python
+---
+
+## Integrate Your {{page.technology}} Application With FusionAuth
+
+{% include docs/integration/_intro.md %}
+
+## Prerequisites
+
+{% include docs/integration/_prerequisites.md %}
+
+## Download and Install FusionAuth
+
+{% include docs/integration/_install-fusionauth.md %}
+
+## Create a User and an API Key
+
+{% include docs/integration/_add-user.md %}
+
+## Configure FusionAuth
+
+Next, you need to set up FusionAuth. This can be done in different ways, but we’re going to use the [{{page.language}} client library](/docs/v1/tech/client-libraries/python). You can use the client library with an IDE of your preference as well.
+
+First, create the required files:
+
+```shell
+touch requirements.txt setup.py
+```
+
+Now, cut and paste the following requirements into `requirements.txt`:
+
+```text
+{% remote_include https://raw.githubusercontent.com/ritza-co/fusionauth-flask-integration/main/requirements.txt %}
+```
+
+Then, copy and paste the following code into the `setup.py` file.
+
+```python
+{% remote_include https://raw.githubusercontent.com/ritza-co/fusionauth-flask-integration/main/setup.py %}
+```
+
+Then, you can run the setup script. Please note that this setup script is designed to run on a newly installed FusionAuth instance with only one user and no tenants other than "Default". To follow this guide on a FusionAuth instance that does not meet these criteria, you may need to modify the above script. Refer to the [Python client library](/docs/v1/tech/client-libraries/python) documentation for more information.
+
+You’ll use a virtual environment `venv` to keep your workspace clean.
+
+```shell
+python -m venv venv && \
+source venv/bin/activate && \
+pip install -r requirements.txt
+```
+
+Now run the setup script, replacing `<your API key>` with the value of the API key noted earlier.
+
+```shell
+fusionauth_api_key=<your API key> python setup.py
+```
+
+If you are using PowerShell, you will need to set the environment variable in a separate command before executing the script.
+
+```shell
+$env:fusionauth_api_key_1='<your API key>'
+python setup.py
+```
+
+This configures FusionAuth for your {{page.technology}} application.
+
+When you’re done, you can log in to your instance and examine the new application configuration, the script created for you.
+
+## Create Your {{page.technology}} Application
+
+Now you are going to create a {{page.technology}} application. While this section builds a simple {{page.technology}} application, you can use the same configuration to integrate your complex {{page.technology}} application with FusionAuth.
+
+First, make a directory:
+
+```shell
+mkdir setup-flask && cd setup-flask
+```
+
+Now, create a new `requirements.txt` file to include {{page.technology}}. Add the `flask`, `python-dotenv`, and `authlib` plugins so that your requirements file looks like this:
+
+```text
```

Comment:
Moved this over to https://github.com/fusionauth/fusionauth-example-python-flask-guide